srcpkgs/virtualbox-ose/template

```diff
@@ -1,28 +1,29 @@
 # Template file for 'virtualbox-ose'
 pkgname=virtualbox-ose
-version=7.0.20
+version=7.1.4
 revision=1
 short_desc="General-purpose full virtualizer for x86 hardware"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only, CDDL-1.0"
 homepage="https://www.virtualbox.org"
 changelog="https://www.virtualbox.org/wiki/Changelog"
 distfiles="http://download.virtualbox.org/virtualbox/${version%[a-z]*}/VirtualBox-${version}.tar.bz2"
-checksum=5cf5979bef66ebab3fcd495796b215a940e8a07c469d4bc56d064de44222dd02
+checksum=872e7a42b41f8558abbf887f1bdc7aac932bb88b2764d07cbce270cab57e3b5e
 
 nopie=yes
 lib32disabled=yes
 archs="x86_64"
 
-hostmakedepends="acpica-utils dev86 perl pkg-config qt5-tools tar yasm which glslang
+hostmakedepends="acpica-utils dev86 perl pkg-config qt6-tools tar yasm which glslang
  vulkan-loader"
 makedepends="device-mapper-devel docbook-xsl gsoap-devel libcap-devel libcurl-devel
- libIDL-devel libvpx-devel libXcomposite-devel libXcursor-devel qt5-tools-devel
- libXinerama-devel libxslt-devel opus-devel pam-devel python3-devel qt5-x11extras-devel
- SDL-devel xorg-server-devel libtpms-devel libxml2-devel libvncserver-devel qt5-devel
+ libIDL-devel libvpx-devel libXcomposite-devel libXcursor-devel qt6-tools-devel
+ libXinerama-devel libxslt-devel opus-devel pam-devel python3-devel qt6-scxml-devel
```

Comment:
is python3-devel needed if we're configuring with --disable-python on line 40?

Reply:
Removing it from makedepends and recompiled packages test local checks indeed. I'll remove it.